content/swagger/resources/categories/routes/categories.yaml

```diff
@@ -124,6 +133,8 @@ get:
               "code": "summer_collection",
               "parent": null,
               "updated": "2021-04-04T09:42:00+02:00",
+              "position": 1,
```

Comment:
Shouldn't the position be 2 here?

Reply:
Nope, because we only set position inside category tree, not between category tree

Reply:
If it's a root category, the level should be at 0 I think